directory/directory_src.go

```diff
@@ -35,7 +36,7 @@ func (s *dirImageSource) GetManifest() ([]byte, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	return m, "", err
+	return m, manifest.GuessMIMEType(m), err
```

Comment:
To me, it really doesn’t make sense for a transport to inspect the input that way, but if we are going by what docker/docker does, https://github.com/docker/distribution/blob/master/manifests.go#L97 is fairly clear that a missing `Content-Type` header is treated as the default mapping.

So, yes, I guess, the docs.